aiven_mysql_migrate/utils.py

```diff
@@ -64,6 +90,9 @@ def _connect(self):
         ssl = None
         if self.ssl:
             ssl = {"require": True}
+
+        LOGGER.debug("connect [%s]- sslca:[%s], sslcert:[%s], sslkey:[%s]", self.name, self.sslca, self.sslcert, self.sslkey)
```

Comment:
Looks a bit too ssl focused that when connecting we log just the name and ssl parameters, IMO we should log all non secret parameters or none.

Reply:
removed